extension/content_dashboard.js

```diff
@@ -7,16 +7,73 @@ $(function() {
     airline = AES.getAirlineCode();
     server = AES.getServerName();
     chrome.storage.local.get(['settings'], function(result) {
-        settings = result.settings;
+      settings = result.settings;
 
-        displayDashboard();
+      if ((settings['flightInformationsFixed'] === undefined) || (settings['flightInformationsFixed'] !== 1)) {
```

Comment:
I thought it might be nice to base this on version number. Perhaps we could do `if versionNumber < 0.6.9`? What do you think?

Reply:
Of course you can do that, but i don't know when exactly the version number changes on internal tests, so i used this simple "versioning" 1,2,3 in case further fixes are necessary and i need to rerun this